cmd/linksharing/.gitignore

```diff
@@ -0,0 +1 @@
+linksharing
```

Comment:
We don't do this in the other `cmd` subpackages, should we also do it?

Reply:
IMO, it helps prevent accidents :)